sdk/servicebus/Microsoft.Azure.ServiceBus/src/Primitives/ConcurrentExpiringSet.cs

```diff
@@ -40,28 +52,38 @@ void ScheduleCleanup()
                 }
 
                 this.cleanupScheduled = true;
-                Task.Run(async () => await this.CollectExpiredEntriesAsync().ConfigureAwait(false));
+                _ = this.CollectExpiredEntriesAsync(token);
```

Comment:
You don't need to assign the result of the method call if you don't plan to access it later (ie. no need for `_ = `)

Reply:
That is technically accurate. Though I consider it best practice to explicitly ignore not consumed tasks. Because if at any time this method would return a task the compiler would start exposing warning. I have to double check the VS diagbostics if they even detect dropped task in void methods